tango_simlib/sim_xmi_parser.py

```diff
@@ -596,6 +663,9 @@ def get_reformatted_properties_metadata(self, property_group):
 
         return properties
 
+    def get_reformatted_class_description_metadata(self):
+        return self.class_description[0]
```

Comment:
The _XMI_ file only has one class description. I was using a list with only one item. I have changed it to a dictionary, and have removed this method as there is no need to convert the `class _description` data structure.